src/main/java/uk/gov/hmcts/reform/roleassignment/domain/model/RoleConfig.java

```diff
@@ -44,20 +41,9 @@ private static void setRoleNameAndCategory(RoleConfigRole role) {
     }
 
     private static RoleConfig buildRoleConfig() {
-        InputStream input = JacksonUtils.class.getClassLoader().getResourceAsStream("role.json");
-        var listType = JacksonUtils.MAPPER.getTypeFactory().constructCollectionType(
-            ArrayList.class,
-            RoleConfigRole.class
-        );
-        List<RoleConfigRole> allRoles = new ArrayList<>();
-        try {
-            allRoles = JacksonUtils.MAPPER.readValue(input, listType);
-        } catch (IOException e) {
-            log.error(e.getMessage());
-        }
+        List<RoleConfigRole> allRoles = JacksonUtils.getRoleConfigs();
```

Comment:
Could you please use static import rather than using class name?

Reply:
It would only help with readability of the code but can be added.